notebooks/build/covid-19-el-paso.ipynb

```diff
@@ -0,0 +1,6756 @@
+{
```

Comment:
<p>I think we ideally want this and the "El Paso specific" distribution together on the same graph</p><p> </p><p>might work to have both of those plus the community distribution all on the same graph</p><p> </p><p>or if not, the community distribution could be on a separate graph</p><p> </p><p>actually what I think is most important is that they're vertically aligned and on the same scale, whether or not they're literally on the same graph doesn't actually matter that much</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/oughtinc/ergo/pull/278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>